doc/configuration/general.md

```diff
@@ -157,6 +157,13 @@ See the section about [redis connection setup](./outgoing-connections.md#redis-s
 
 When a user's session is replaced (due to a full JID conflict) by a new one, this parameter specifies the time MongooseIM waits for the old sessions to close. The default value is sufficient in most cases. If you observe `replaced_wait_timeout` warning in logs, then most probably the old sessions are frozen for some reason and it should be investigated.
 
+### `general.max_users_per_domain`
```

Comment:
I think this is in the wrong section - it is user account management, not session management.

Just a question: why not place it in the `auth` section which has all options related to user accounts?

Reply:
I’ve chosen to include it in the `general` section as I believed it was more connected with the domains themselves rather than user authentication. However, if it would be more appropriate, I can move it to the `auth` section.

Reply:
Actually it only limits certain authentication methods, because some of them (`external`, `http`, `jwt`, `pki`, ...) don't create users the usual way, so I think it's best placed in the `auth` section - not to give the impression that it works for all methods.